xml/System.Reflection/MetadataLoadContext.xml

```diff
@@ -60,9 +68,57 @@
         <ReturnType>System.Reflection.Assembly</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the assembly that denotes the "system assembly" that houses the well-known types such as <see cref="T:System.Int32" />.</summary>
+        <value>An <see cref="T:System.Reflection.Assembly" /> instance.</value>
+        <remarks>
+          <format type="text/markdown">
+            <![CDATA[
+## Remarks
+
+The core assembly is treated differently than other assemblies because references to these well-known types do not include the assembly reference, unlike normal types.
+            
+Typically, this assembly is named "mscorlib", or "netstandard". If the core assembly cannot be found, the value will be null and many other reflection methods, including those that parse method signatures, will throw.
+            
+The <xref:System.Reflection.MetadataLoadContext.CoreAssembly> is determined by passing the coreAssemblyName parameter passed to the <xref:System.Reflection.MetadataAssemblyResolver> constructor
+to the MetadataAssemblyResolver's <xref:System.Reflection.MetadataAssemblyResolver.Resolve> method.
```

Comment:
```suggestion
to the <xref:System.Reflection.MetadataAssemblyResolver.Resolve?displayProperty=nameWithType> method.
```

Reply:
Ooooh nice suggestion. Didn't know this was possible.